korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.2.0-preview2-20181016.3
-commithash:17ee7986dbe75408d9e4efd94eab1003adcc3ad7
+version:2.2.0-preview2-20181019.5

```

Comment:
nit: Why is korebuild still at preview2?

Reply:
we never changed its version in aspnet/BuildTools. The pre-release labeling on build tools is sort of meaningless (shrug)